### ISSUE TITLE
Outline View state is not memorized between sessions

### ISSUE THREAD

**Issue by ffxsam** [facebook/nuclide#725]
### Issue and Steps to Reproduce
Outline View state (open/closed) is not memorized across sessions.

### Expected Behavior
I'd expect that if I quit Atom and re-open the same project, that the Outline View will either be visible or hidden based on its state when I quit Atom.

### Actual Behavior
Outline View is always hidden, and if I open it, its width is not memorized either.

### Versions
* Atom: 1.10.2
* Nuclide: 0.170.0
* Client OS: Mac OS 10.11.6
* Server OS (optional):

### Additional Details
* Installed packages (`apm ls --installed`):

```
├── autocomplete-modules@1.6.3
├── expose@0.12.1
├── git-time-machine@1.5.3
├── goto-definition@1.1.19
├── haskell-grammar@0.4.0
├── highlight-selected@0.11.2
├── js-refactor@0.7.3
├── language-babel@2.41.0
├── language-ini@1.16.0
├── language-ocaml@1.1.2
├── language-swift@0.4.0
├── language-thrift@1.0.2
├── linter@1.11.16
├── linter-eslint@7.3.2
├── merge-conflicts@1.4.4
├── minimap@4.25.0
├── nuclide@0.170.0
├── nuclide-test-runner-jest@0.3.0
├── refactor@0.9.2
├── sort-lines@0.14.0
├── todo-show@1.7.0
└── tool-bar@1.0.1
```

**Comment by nmote** [comment]
@matthewwithanm I can reproduce this. Related to the migration to workspace views? It definitely used to work.

**Comment by matthewwithanm** [comment]
Yeah, this is a result of how package serialization interacts with deactivation. I'll look into it.

**Comment by matthewwithanm** [comment]
Thanks for the report @ffxsam! This will be fixed once atom/atom#12759 lands so I'm going to close.
